soc_023_rw1_fragile_states_index/README.md

```diff
@@ -0,0 +1,13 @@
+This file describes the data pre-processing that was done to [Fragile States Index](https://fragilestatesindex.org/) for [display on Resource Watch](https://bit.ly/2O6Qv4F).
+
+The source provided this dataset as a set of xlsx files accessed through its [website](https://fragilestatesindex.org/excel/). 
+
+The xlsx files were concatenated into a sole dataframe to process them.
```

Comment:
```suggestion
The excel files were concatenated into a sole pandas dataframe. Column headers were renamed to remove special characters and spaces so that the table can be uploaded to Carto without losing information.
```